### ISSUE TITLE
Crash when highlighting

### ISSUE THREAD

**Issue by jouki** [DarkShadow44/resourcehighlighter-dark#9]
Hi,

I've got another crash when I tried to highlight a resource. This time I'm not able to replicate the issue and have a screenshot of crashlog only. I've noticed another issue and may be related. #5 
<img width="425" alt="Screenshot_94" src="https://user-images.githubusercontent.com/1493141/128898627-44071cb6-6694-4aac-9a0d-5cdfcaced227.png">

Would be possible to release a new version mentioned in #5 so it can be conveniently downloaded from mod manager? 🙂

Cheers,
Jouki

**Comment by DarkShadow44** [comment]
Not sure how that could happen... Anyways, maybe it's fixed with the latest change. If you can reproduce, please tell me how.

**Comment by jouki** [comment]
I think I was able to replicate the issue.
1. Go to another planet via satellite mode from Space Exploration(N)
2. SHIFT + H (Show highlight window)
3. Highlight something (Iron Ore)
4. Then go back (disable the satellite mode you were in)
5. Close the highlight window
6. I think I went to another planet (Nauvis as the main planet), but this step may not be important
7. Again open highlight window
5. Repeat step 3 -> Crash
